src/util/jemalloc.rs

```diff
@@ -25,6 +25,7 @@ mod jemalloc {
         );
     }
 
+    #[cfg_attr(feature = "cargo-clippy", allow(cast_ptr_alignment))]
```

Comment:
Is this necessary?

Reply:
Yes, clippy blames https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/tikv_ghpr_test/detail/tikv_ghpr_test/1408/pipeline .